applets/006-tcpdump/applet.py

```diff
@@ -45,81 +45,118 @@ def tcpdump_start(x, arg_d):
         time.sleep(1)
 
     if no_pid_found:
-        x.p.msg("error: tcpdump pid on host {} could not be "
+        x.p.err("error: tcpdump pid on host {} could not be "
                 "retrieved\n".format(arg_d["host_name"]))
         return False
 
     # save pid to file
     path_to_tcpdump_pid = "/tmp/net-applet-shuffler/tcpdump_{}"\
                           .format(arg_d["applet_id"])
-    _, _, exit_code_1 = x.ssh.exec(arg_d["host_ip_control"], arg_d["host_user"],
-                                   "touch {}".format(path_to_tcpdump_pid))
-    _, _, exit_code_2 = x.ssh.exec(arg_d["host_ip_control"], arg_d["host_user"],
-                                   "sh -c \"echo '{}' > {}\""
-                                   .format(pid_tcpdump, path_to_tcpdump_pid))
+    exit_code_1 = x.ssh.exec(arg_d["host_ip_control"], arg_d["host_user"],
+                             "touch {}".format(path_to_tcpdump_pid))
+    exit_code_2 = x.ssh.exec(arg_d["host_ip_control"], arg_d["host_user"],
+                             "sh -c \"echo '{}' > {}\""
+                             .format(pid_tcpdump, path_to_tcpdump_pid))
     if (exit_code_1 or exit_code_2) != 0:
```

Comment:
Awesome ASCII skills and nice explanation. Somehow I didn't think about that at all. I will fix it to the "Or better:" solution, since it is clear and consistent.